doc/cookbook/source/examples/classifier/kmeans.rst

```diff
@@ -0,0 +1,47 @@
+==================
+:math:`k`-means clustering
+==================
+:math:`k`-means clustering aims to partition :math:`n` observations into :math:`k` (:math:`\leq n`) clusters in which each observation belongs to the cluster with the nearest mean, serving as a prototype of the cluster.
+
+The :math:`n` observations are represented by :math:`n` :math:`d`-dimensional real vecoters, :math:`\mathbf{x} = (x_1, x_2, ..., x_n)`.
+
+
+
+In :math:`k`-means clustering, the :math:`n` observations will partitioned into :math:`k` (:math:`\leq n`) sets :math:`\mathbf{S} = {S_1, S_2, ..., S_k}`, with minimal within-cluster sum of squares (WCSS) (sum of distance functions of each point in the cluster to the :math:`k^{th}` center). 
```

Comment:
Try to be a bit more concise (that is: less text) here. Just stating the objective should be fine